frontend/flutter/lib/splash.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+
+import 'main.dart';
+
+class Splash extends StatefulWidget {
+  const Splash({ Key? key }) : super(key: key);
+
+  @override
+  State<Splash> createState() => _SplashState();
+}
+
+class _SplashState extends State<Splash> {
+  @override
+  void initState() {
+    super.initState();
+    _naviagteToHome();
+  }
+
+
+  _naviagteToHome()async{
+    await Future.delayed(Duration(milliseconds: 1700), () {});
+    Navigator.pushReplacement(context, MaterialPageRoute(builder: (context)=>App().getHomePage()));
```

Comment:
Feel free to move the `getHomePage` function in this file. Also move the RolePage widget.